ntbs-service/Models/SocialRiskFactors.cs

```diff
@@ -1,15 +1,16 @@
 using Microsoft.EntityFrameworkCore;
 using ntbs_service.Models.Enums;
+using ntbs_service.Models.Factories;
 
 namespace ntbs_service.Models
 {
     [Owned]
     public class SocialRiskFactors : ModelBase
     {
         public SocialRiskFactors() {
-            RiskFactorHomelessness = new RiskFactorDetails() { Type = RiskFactorType.Homelessness };
-            RiskFactorDrugs = new RiskFactorDetails() { Type = RiskFactorType.Drugs };
-            RiskFactorImprisonment = new RiskFactorDetails() { Type = RiskFactorType.Imprisonment };
+            RiskFactorHomelessness = RiskFactorFactory.RiskFactorHomelessness();

```

Comment:
That's looking much nicer!
I wonder if these methods would be more discoverable as statics on their class? (e.g. `RiskFactoryDeatils.Homelessness()`)
Happy with either way :)

Reply:
Ooh, yet another would be actually to just provide another constructor which requires the type attribtue. The benefit of that is that it can get used with initializer if needed. so all these would be valid:
```
var RiskFactorHomelessness = new RiskFactorDeatils(RiskFactorType.Homelessness);
varRiskFactorHomelessness = new RiskFactorDeatils(RiskFactorType.Homelessness) {
  IsCurrent = true
};
var RiskFactorHomelessness = new RiskFactorDeatils {
  Type = RiskFactorType.Homelessness
  IsCurrent = true
};
```

Reply:
That might be the simplest option :)